Godeps/Godeps.json

```diff
@@ -341,6 +341,11 @@
 			"Rev": "35f3a090b2f7ebe32daf0c6a73eb060c669e2128"
 		},
 		{
+			"ImportPath": "github.com/containernetworking/cni/pkg/utils/sysctl",
+			"Comment": "v0.3.0-rc2",
+			"Rev": "6f63d9d7077097b13dc317f749421316b6f91f86"
+		},
```

Comment:
Isn't this commit missing all the package files in the workspace? (and maybe then split it in its own commit)